lib/rules/one-var.js

```diff
@@ -53,6 +84,34 @@ module.exports = function(context) {
         }
     }
 
+    /**
+     * Determines if there is more than one let statement in the current scope.
+     * @returns {boolean} Returns true if it is the first let declaration, false if not.
```

Comment:
I see this is copied from the original `hasOnlyOneVar` implementation. Can you confirm that `hasOnlyOneVar` never actually did what it claimed to do? I see below that if this returns true, indicating the block has a single variable declaration of a given type, then it will warn to combine with a previous variable declaration.